src/Components/Admin/EnvironmentEditor/environmentBar.js

```diff
@@ -74,22 +101,42 @@ export default function EnvironmentBar({
             <AppBar position="fixed" className={classes.appBar} elevation={0}>
                 <Toolbar className={classes.toolbar}>
                     <div className={classes.buttonWrapperLeft}>
-                        <IconButton
+                        <Button
                             aria-label="environment-menu"
                             color="inherit"
+                            onClick={onMenuClick}
                         >
                             <MenuIcon />
-                        </IconButton>
+                        </Button>
+
+                        <Menu
+                            anchorEl={menuAnchorEl}
+                            anchorOrigin={{
+                                vertical: "top",
+                                horizontal: "center",
+                            }}
+                            transformOrigin={{
+                                vertical: "top",
+                                horizontal: "center",
+                            }}
+                            keepMounted
+                            open={menuOpen}
+                            onClose={onMenuClose}
+                        >
+                            <MenuItem>Menu</MenuItem>
```

Comment:
I know the design had this, but it shouldn't be a menu item. It's more of a title. Can you try rendering this as a title (perhaps a <h2> tag) or look into how you can render a title within the dropdown menu for material-ui Menu component? This is for sure not what we want since that will be a clickable menu item. Same with "New Scene" in the other dropdown.

Reply:
I thought you might comment on that - sure, I'll look into it. Is it that we don't want the animation that shows that it is a clickable item? Because other than that, it's pretty un-interactable

Reply:
Yeah it has slightly different styling (it appears grey on Figma) and yeah, no animation.